src/quantum/config.js

```diff
@@ -62,4 +62,40 @@ const CONFIG = {
 
 };
 
-export default CONFIG;
+const PAGES = {
+  header:
+    ['title', 'label', 'frameworkId', 'platform', 'option', 'project', 'suite', 'extraOptions'],
```

Comment:
You can probably add one more value to identify on which page to show up (e.g. 32-bit vs 64-bit pages).
We can probably call the propery 'page' or 'viewPlatform'

Reply:
yes

Reply:
...64 and 32